.github/workflows/build-package.yml

```diff
@@ -0,0 +1,62 @@
+name: Build Package
+
+env:
+  MAIN_BRANCH: "all-pgautofailover"
+  PACKAGE_CLOUD_REPO_NAME: "sample"
+  PACKAGE_ENCRYPTION_KEY: ${{ secrets.PACKAGE_ENCRYPTION_KEY }}
+on:
+  push:
+    branches: "**"
+
+  workflow_dispatch:
+
+jobs:
+  build_package:
+    name: Build package
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - el/7
+          - el/8
+          - ol/7
+          - debian/stretch
+          - debian/buster
+          - ubuntu/xenial
```

Comment:
I think we should remove this line

```suggestion
```

Reply:
similarly we can update `.travis.yml`, preferably in a separate PR.

Reply:
Asking @DimCitus might be useful